app/views/users/_form.haml

```diff
@@ -11,7 +11,7 @@
     = f.text_field :email, required: true
   .form-group.row
     .col-md-2= f.label :spire
-    = f.text_field :spire, required: true
+    = f.text_field :spire, required: true, placeholder: 'Spire-Id@umass.edu'
```

Comment:
I usually put examples in placeholders.

`placeholder: 'e.g. 08675309@umass.edu'`